bin/gpo

```diff
@@ -170,15 +169,17 @@ class gPodderCli(object):
         self._model = self.core.model
 
         self._current_action = ''
-        self._commands = dict((name.rstrip('_'), func)
+        self._commands = dict(
+            (name.rstrip('_'), func)
```

Comment:
why not keep it on same line?

Reply:
Arguments on first line forbidden when not using vertical alignment.
https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
My bad, I didn't see that the line was continued. It's ok as it is.